courses/machine_learning/deepdive2/structured/solutions/3a_bqml_baseline_babyweight.ipynb

```diff
@@ -0,0 +1,616 @@
+{
```

Comment:
<p> </p><ul><li>"In this lab, we will create a baseline model with BQML," -&gt; "Then we will" </li><li>"and calculate the RMSE of our baseline model." -&gt; "and calculate its RMSE."</li><li>Remove "Each learning objective will correspond to a&nbsp;<strong>#TODO</strong>&nbsp;in this student lab notebook -- try to complete this notebook first and then review the&nbsp;<a href="https://github.com/GoogleCloudPlatform/training-data-analyst/raw/e583c2c028f8a123e99b6f4954ceae24acd709d3/courses/machine_learning/deepdive2/structured/solutions/3a_bqml_baseline_babyweight.ipynb" target="_blank">solution notebook</a>." since this is the solution notebook</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/668/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Done</p>